charts/fluentd-elasticsearch/Chart.yaml

```diff
@@ -1,5 +1,5 @@
 name: fluentd-elasticsearch
-version: 2.8.3
+version: 2.8.4
```

Comment:
As a new feature is added we should go to 2.9.0

Reply:
Updated to 2.9.0